playwright/core/expectScreenshotFixture.ts

```diff
@@ -10,6 +10,7 @@ export const expectScreenshotFixture: PlaywrightFixture<ExpectScreenshotFixture>
     const expectScreenshot: ExpectScreenshotFixture = async ({
         component,
         screenshotName,
+        screenshotPostfix,
```

Comment:
For multiple screenshots per test
`
 test(......... () => {
      await expectScreenshot();

      //...

      await expectScreenshot({
          screenshotPostfix: 'after hover',
      });
});
`

Reply:
Maybe `screenshotNameSuffix`?